web-common/src/components/icons/connectors/MotherDuckIcon.svelte

```diff
@@ -0,0 +1,34 @@
SoLd1ySBxAwVCHG5oYDNfbevEqZGuopixY8plLMa9aslccff+z2vCO6RMvh/T+Kr5+frN91XKaOWSvv/mOYvPbMRKXoOi5YLt3Bdcb+98N1In4Shw9SP52JJEOgvP7Fennv67PSo7+f9AhsJ5FInTnBhy9HHv/arXKJRGJy+th+yMt54ETVhXokGu19fHzOteQ4J2tqjl7k4QkT/saKld+tYsRmmv3oUHFGxoQ7Fdxi8gLSCUj1dEei4MNnxyukt27dppAeMWKEvP3227Jz505JSjylkD506qJMXX5MPGYEyebzN6Smth75cVdEVPZqc6HJqnLEuwO9PKbAjS6OasmczMRROjeDnMURkK9/zJCXvj0lz/1wXooramQWgpl5vxpLpLVz5841EGn2juX7TmuOs/72Wo+Li9uFBqMSz5xp7Nmjh9o7HzB/tXTEvncWD9sgIDn99lTp1S1cmLpdv269PP30bAkPD4OhvK03ZX9CsvxmyT5xwgJNya6ULS+Nkgkj+qgTVCNfW97klUFZ3YvjnCwRL4VGP/7BLJVSvlVcIQVFpSoJYgeni+aUaS07WxsqDQuSDtrq1Wv0v/3tM4rJUOL/hbN8f6NotwZzh+ajkDhi+QoQH4bDA3ZHDh9u6B/X2S7hk+ny9or90t1LLwufHKaQTktL0zZs2KD74P14sUUAcjOvUHlSqbCzG7Cxt2vfTZEO8OPhou54dZyMGdJDjcnggiclzErttZ5GG8/AvA6ZlQ5YGk4IQyklbtAxPp4hXJ5wD6ncLVpZWbku9cZNm5MnT9ngIJPqqGOnTjdg8+cC6e083tYW4sLzn3DxHHbs2JEEZfEizrB+MXfuc3Zf/v3LOoSE9ttiIpWkMHaBCTR1i+mhe/53zxJpslmXinNsu0+mShqCBh9M8s05PSSiva8M798N5+HcpAxZETecjrRFZFZ/j2SEmnXzD9OIJVCycUaj4mxOdrY2fNQ43cNjR9vyqGdVVZVcv35dtm7Z1LJZDnBZCRP9HiormW/DmT51zq3lRy3vdaCMngf/YPQ/Rjw7v3efPhL/XnxjbEzT6QnswNi89tprhsRElbgUnI9rnDJ5shkb8HQiKFqKQM2dUvQaGRouXbbC8GH8OwKnT2L+sFjcQADjPUSdjZkNOoG1PL93iHw87zENykqHdctXTMNSW5PrFSg5mHMa9NVx3O8CDlAEChzwi+2Wn05MvWn1c3u9g2ovgmpv4r2x1TeVkIp/QCqGoL7PY48/LmNGj+E6x6lIDwtdytq6Osm8dt3m1MlThvfefQefGbT6ugopr27QdfvDEiQhbbDDe/c13qTT9HIMVuTPE2PkD7PGmo4eO2Y7eNCg6jfeeOMxWJ5EHOpxx7Vg7ty5PFR8G+ifwychUUgcBW2KuvUlrnRoFPJA+lMcid6ZlZU1FfddWQ+EU3E8bOPq1asTocl9IWoLkfGYghKA97QGrYHcKB056sEQ5tvgzGiIb3Rk2f0AnRd4L+LnqbZ9cXhYMbIGiJ2eOXPmLbwtaNGPLThuYKqp2T9v8QoJ2Z88tf2gkAfHdUuXLmXS6y3rZ+CyuqWygDhx0BdwSvhjhIKxqZcuRZ6/eBHpWjF4erpXd+4clfvSG29smTRq1Lj8gsJFbAinwgKXl4S9J9Dvo9cH1S++nk0nvvNw2hnQiH0/aE4kX8LCXHl+HsuJtDQDaeWT8F1ruB/E2cYMLuuoGKAVrRqU9awzAGkT81kYsBGIZ6Oe5Qd+QCguLpVjx47JMXhUgBIe1VKI6PVwXJCIuC+tzmQI8MH3NFeE6zdu8FILK8Q1Ls1I3xZn1t0JWjswd/qO9VrzWXZSk1wi0XTNdRYijWd9c0JPvcNzS3BsftAVFBYjt9605cScG5h4TyCzGZLSJDIEpSnDyWi2q5g0aRKynFA+RuN99MQvmzRv0939/7Jzin9LzltbWwlgFTES1qrdrZOqzL+ZiQQhjl8ZbDUjkhEtAx9rR62v7ISp5hh4inRSqquqdTt27+NnxVgCisuQSusYrZv/7Pl/wvGfNb5HBSdBTcrSckLkDvDGUU1ofBfsbuJ0PqruDHzLbet8ppm9jTwjr5WVl+kaasolIiIit7klpYxj3Rf8v0S8zQkgFi7Hi9o6/FkHXH7NA2njEsg6FfadgHRh6qsAbnKon1Js2q1bhWrusCrpbId1TsTvTsEWA/ySiKtJDenTpwTjVyBm5jQ0T2zz1kOE74o58FEHFRAgdQrwUNPPbzq3g79t8U9mBSTobrRTbVr+/JKIKzH8zXPP4cyXFJWUNPkYftz9AOJ3mjXrqfwU4vDTw+D6AjTYZl4Lo6KiuB9AjU69c9/wiyEOW68Qx18HcY2nZ2fncJL6sGBvtQ3FxCWVdksCUESY4WGOrZHY49xc59BgWEKLbt/+/Wx/9vXXX2dHNrAu92XG2IjwiyFOnx/jqXgY1+NwNzm+bb/YCOao1V48bTr/xIPhKZHmrqsvTkj6IQI7mFggfxvTDXtiRklNS9Nt2byZG/1r2Ak8NHqJbHLf8Ish3jwjxVBwf+9nf/1LRXpGBk8kWZZM7SUHTuarHRemsvxRmOggJntvVcvmxDx5flKk/GryUHZj/nbZcvaTDs9xGSu4383rfyzAqyOhrdK868X58+GMaSYc49De/cdGTUa+pcmDLcq0D7RZb3yjbdqdoOETQiPifvoI2oABAyYS0eYNAt7+Z4M1tQVvawxmqi1d+i0RQmJF05CT1zYAyTXbj2oHEpK1zJv5Grw1BRaLuXbZsuXUE5qDg8N7zVjShP1/A+S4mjBCyC9xr33yyaem0pIS/oVhM5oKV/K4oaSosG779u3myY9OpeSzvIFiBav0WJ//s6+M5qwzdHd3/wr3CiloaPxJ5zIL/vhW++yzz7Snn55jRZbXg0g6DLa2w/WX1k8thv5f3Davd9VDt27dKPY7UOjVWZFlsM1tn9dDQkLicG0J/2uk/xukTwoO7q98bAAAAABJRU5ErkJggg=="
```

Comment:
Can we use an actual SVG – like you do for Athena – rather than embed a PNG like this? (Same comment applies to MySQL, Postgres, and Redshift.)